### PR TITLE
✂️ Separate docker test case

### PR DESCRIPTION
**Pull request by trajepl**
## Describe your changes
Separate docker test case to avoid hanging on CI pipeline.

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Format your code by running `pre-commit run --all-files`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

File: examples/test/utils.py [hunk 1]
```diff
@@ -51,6 +51,9 @@ def patch_config(config_json_path: str, search_algorithm: str, execution_order:
         # set docker_system
         set_docker_system(olive_config)
         olive_config["engine"]["target"] = system
+        # reduce agent size for docker system
+        olive_config["passes"]["transformers_optimization"]["disable_search"] = True
+        olive_config["engine"]["search_strategy"]["search_algorithm_config"]["num_samples"] = 2
```
Comment (guotuofeng) [inline examples/test/utils.py:56]: add comments here to describe the issue

Comment (trajepl) [reply examples/test/utils.py:60]: Updated.

File: examples/test/test_bert_ptq_cpu.py [hunk 2]
```diff
@@ -22,12 +21,9 @@ def setup():
 
 @pytest.mark.parametrize("search_algorithm", ["tpe"])
 @pytest.mark.parametrize("execution_order", ["joint"])
-@pytest.mark.parametrize("system", ["local_system", "aml_system", "docker_system"])
+@pytest.mark.parametrize("system", ["local_system"])
```
[the remark this answers is not in the document]
Comment (jambayk) [reply examples/test/test_bert_ptq_cpu.py:24]: Created this PR with the change since @trajepl is unavailable currently

**Comment by jambayk** [comment]
Closing since #470 has been merges with the same changes.

**Pull request closed without merging**